pull_request_template

```diff
@@ -7,8 +7,10 @@ Before creating this pull request, please make sure you have a separate ticket f
 
 
 ## Checklist
-- [ ] Make sure you change all affected versions (e.g. 1.10, 1.11, 1.12, 1.13).
-- [ ] Test all commands and procedures where applicable.
+- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
+- [ ] Test all commands and procedures, if applicable.
+- [ ] Create your PR agains Staging, not Master. 
+- [ ] Provide an estimated date for deploying the doc change. Note: Improvements or fixes can be merged ASAP. 
```

Comment:
This is interesting... I can see the convenience of having it in the description but shouldn't this info be reflected in the jira tickets? I assume the goal is to let you know when/not to merge? Perhaps this can be solved with the `ready to merge` label instead?

Reply:
@samvantran  maybe so with the label, but people tend to put the readytomerge label on too soon, before all reviews are done. Yes, the goal is that I have to go around asking "hey is this ready to go out yet" and many times PRs sit out here for weeks waiting for... I have no idea why. If we need to think through this a bit more, I can take this line out and we can discuss further.

Reply:
makes sense. sounds like this would make your life a little easier as others are abusing labels/not staying on top of things.